TheChocolateRiversDB/4 data.sql

```diff
@@ -0,0 +1,67 @@
+go
+use ChocolateRiverDB
+go
+delete Chocolate
+go
+
+insert Chocolate (ChocolateType, Flavor, Shape, RecipeSource, YearPutonMarket, ChocolateWeight, DateSold)
+select 
+--ChocolateType
+case  m.Medal 
+when 'gold' then 1
+when 'silver' then 2
+when 'bronze' then 3
+end,
+--Flavor
+case
+when m.SportSubCategory like '%ladies%' or m.SportSubCategory like '%women%' then 1
+when m.SportSubCategory like '%men%' then 2
+when m.SportSubcategory like '%[0-9]%' then 3
+when m.SportSubcategory like '% %' then 5
```

Comment:
-- LL For the fourth when in the case statement:
1. This should be for the 'Dark Salted Caramel Flavor' (4), the else should be for 'Milk Creme and Dark Creme' (5).
2. Besides for checking that it's only one word, you want to check that the SportSubcategory is not like any of the first three conditions.

Reply:
How do I check if it's only one word?